web/cypress/integration/route/create-route-with-cors-form.spec.js

```diff
@@ -62,7 +64,7 @@ context('Create and delete route with cors form', () => {
     cy.get(selector.nodes_0_port).clear().type(data.port);
     cy.get(selector.nodes_0_weight).clear().type(data.weight);
     cy.contains('Next').click();
-
+    cy.wait(1000);
```

Comment:
IMO, We refuse to abuse `wait`, it wastes a lot of our time, and we don't use it except as a last resort. Where we can use some assertions to ensure that the test is stable.

Reply:
OK, I've fixed it, you can check it.